src/main/java/de/tum/in/test/api/ast/asserting/UnwantedNodesAssert.java

```diff
@@ -0,0 +1,124 @@
+package de.tum.in.test.api.ast.asserting;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
+import org.assertj.core.api.AbstractAssert;
+
+import com.github.javaparser.ParserConfiguration.LanguageLevel;
+import com.github.javaparser.StaticJavaParser;
+
+import de.tum.in.test.api.ast.model.UnwantedNode;
+import de.tum.in.test.api.ast.type.*;
+
+/**
+ * Checks whole Java-Files for unwanted nodes
+ */
+public class UnwantedNodesAssert extends AbstractAssert<UnwantedNodesAssert, Path> {
+
+	/**
+	 * The flag, determining whether the path points to a file or to a directory
```

Comment:
```suggestion
	 * true when the path points to a directory, false if it points to a file
```

Reply:
[Implemented part of the new feedback](https://github.com/ls1intum/Ares/pull/248/commits/ead5d75f0814dbd3738c57b058de8759b980e46d): Changed the comment.